iot/iot-tenant-cleaner/src/main/resources/logback.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!DOCTYPE xml>
+
+<configuration>
+
+  <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+    <encoder>
+      <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
```

Comment:
We should standardize on the log format for all components. This just caught my attention now, so maybe its a separate PR to fix in other components as well. I believe what we have standardized on (for both java and node at least), is:

```
<pattern>%d{yyyy-MM-dd'T'HH:mm:ss.SSS'Z',GMT} %-5p [%c{0}] %m%n</pattern>
```

Reply:
Yes, I only have that in there for debugging. Not having the thread name can be a problem. I will remove that.